src/plr.h

```diff
@@ -311,6 +311,75 @@ extern void R_RunExitFinalizers(void);
 	HeapTuple		tup; \
 	HeapTupleHeader	dnewtup; \
 	HeapTupleHeader	dtrigtup
+
+#if GP_VERSION_NUM >= 70000
+#define SET_INSERT_ARGS_567 \
+	do { \
+		nullablearg[5].value = DirectFunctionCall1(textin, CStringGetDatum("INSERT")); \
```

Comment:
```suggestion
		nullablearg[5].value = CStringGetTextDatum("INSERT"); \
```

I think we can use `CStringGetTextDatum()` here.

Looks that we're using `DirectFunctionCall1(textin, ...)` here and there. Let's keep it as is....